rpa-recorder/6.4/modules/ROOT/pages/user-interface.adoc

```diff
@@ -27,6 +28,59 @@ The *Recording Toolbar* appears on the screen when RPA Recorder starts recording
 
 When a recording state changes, a toast notification appears indicating current state. The notification stays for two seconds and disappears automatically.
 
+== Image-Based Mouse Clicks
+
+Use image-based mouse clicks to record mouse clicks on a UI element that otherwise cannot be recorded properly. The feature guides you through the mouse click process. An image on the screen and mouse clicks must be captured, and the mouse clicks are recorded relative to the captured image.
+
+Use this feature at any point during a normal recording session. The feature also checks your display settings automatically while recording to ensure the feature records correctly.
+
+This feature will result in _Search Pattern_ and _Mouse Action_ Action Steps when the workflow is generated in RPA Builder from the recording.
```

Comment:
Again - I am confused. Here you say it results in _Search Pattern_ and _Mouse Action_ Action Steps and then you say it is available for _Search Pattern_ and _Mouse Action_ Action Steps.